### DISCUSSION TITLE
下载测速队列里的数字是什么意思？

### DISCUSSION THREAD

**Discussion by Idealisten** [XIU2/CloudflareSpeedTest#433] [提问]
下载测速队列里的数字是所有PING通的IP里延迟符合要求的IP数量吗？
为啥我这2179个可用但是下载队列只有8个？我的延迟设置的很宽松了呀
![](https://picture.runtofree.link/picture/2023/09/3376ffed18cca50d4ace901327170135.png)

**Comment by charSLee013** [comment]
**2179**可用指的是：在超时1000ms之内能TCP连接成功的IP数量

在延迟测速之后，会进行 **延迟**+**丢包率**的过滤
看图设置的条件是：平均延迟上限是250ms 和 平均延迟下限是 40ms

在**过滤**完毕之后，才进行下载测速，所以下载测速队列里面只有8个**符合过滤条件**的IP地址

**Comment by XIU2** [comment]
如楼上所说，可用：2179 是延迟测速通过的 IP 数量（即没有超时的）。
下载测速队列：8 指的是在这 2179 个 IP 结果中，延迟符合 40~250ms 的 IP 数量，也就是只有 8 个 IP 的延迟测速结果在该延迟范围内。
